mir-ci/mir_ci/test_drag_and_drop.py

```diff
@@ -30,7 +34,15 @@
         apps.mir_demo_server(),
     ],
 )
-@pytest.mark.deps(debs=("libgtk-4-dev",), pip_pkgs=(("pygobject", "gi"), ("robotframework~=6.1.1", "robot")))
+@pytest.mark.deps(
+    debs=("libgtk-4-dev",),
+    pip_pkgs=(
+        ("pygobject", "gi"),
+        ("robotframework~=6.1.1", "robot"),
+        ("rpaframework", "rpa"),
+        ("rpaframework-recognition", "rpa-rec"),
```

Comment:
So the format here is `(<package>, <module>)`, where the package is what gets `pip install`ed, and `<module>` is what gets confirmed exists.
```suggestion
        ("rpaframework", "RPA"),
        ("rpaframework-recognition", "RPA.recognition"),
```

The idea is to avoid installing things multiple times.

Reply:
Oh, that makes sense. I assumed it was just a short name for the package.